objects.go

```diff
@@ -604,7 +655,7 @@ type HostRecordIpv4Addr struct {
 	IBBase     `json:"-"`
 	Ipv4Addr   string `json:"ipv4addr,omitempty"`
 	Ref        string `json:"_ref,omitempty"`
-	Mac        string `json:"mac"`
+	Mac        string `json:"mac,omitempty"`
```

Comment:
Why this is needed? We had issues with making MAC-address field empty (deleting association), that is why this 'omitempty' option was removed.

Reply:
This was a new change that i did not have in my branch. I have removed it in my branch.